daemon/containerd/image_list.go

```diff
@@ -91,12 +91,25 @@ func (i *ImageService) Images(ctx context.Context, opts types.ImageListOptions)
 			return nil, err
 		}
 
+		target := img.Target().Digest
+
+		ref, err := reference.ParseNamed(img.Name())
+		digests := []string{"<none>@<none>"}
+		tags := []string{"<none>:<none>"}
```

Comment:
I'd rather declare these two up above with `target` to make the intention of this block clearer. Also, as we never use these values as arrays, I'd rather declare the literal strings up top, and preserve the array-wrapping on lines 111 and 112.

Reply:
Done, thanks!